homeassistant/components/nissan_leaf/button.py

```diff
@@ -0,0 +1,51 @@
+"""Button to start charging the Nissan Leaf."""
+from __future__ import annotations
+
+import logging
+
+from homeassistant.components.button import ButtonEntity
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
+
+from . import DATA_CHARGING, DATA_LEAF, LeafEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
+    """Set up of a Nissan Leaf button."""
+    if discovery_info is None:
+        return
+
+    devices: list[LeafEntity] = []
```

Comment:
Please rename this to `entities`.

Reply:
Hi @MartinHjelmare, no problems. I'll do this in a separate PR based on https://github.com/filcole/core/tree/nissanleaf_devicestoentities because there's other places in the nissan leaf integration with this issue.

Reply:
Thanks!